tests/bud.bats

```diff
@@ -3180,6 +3180,12 @@ _EOF
   assert "$output" !~ '--build-arg NEWSECRET=<VALUE>'
 }
 
+@test "bud with arg in from statement" {
+  _prefetch alpine
+  run_buildah build -t testbud $WITH_POLICY_JSON --build-arg app_type=x --file $BUDFILES/with-arg/Dockerfilefromarg .
+  expect_output --substring 'world'
+}
+
```

Comment:
Just in case, could you add a test with multiple build-args?

Reply:
Sure. I'll amend this.